### PR TITLE
add, delete area

### PR DESCRIPTION
**Pull request by at-tuannd**

### CODE REVIEW

File: app/Http/Controllers/Admin/AreaController.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id [description]
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        if (Auth::user()->level==1) {
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/AreaController.php]: Please use config you already define

File: app/Http/Requests/AreaRequest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Get the validation messages that apply to the request.
+     *
+     * @return array
+     */
+    public function messages()
+    {
+        return [
+            'namearea.required'=>'please enter area name!',
```
Comment (at-vietpb) [inline app/Http/Requests/AreaRequest.php]: Define message to lang file

File: app/Http/Controllers/Admin/AreaController.php (hunk 1 above)
Comment (at-tuannd) [inline app/Http/Controllers/Admin/AreaController.php]: yes

File: app/Http/Requests/AreaRequest.php (hunk 2 above)
Comment (at-tuannd) [inline app/Http/Requests/AreaRequest.php]: yes

**Pull request merged**